lib/config.js

```diff
@@ -193,14 +208,20 @@ function validateEnvConfig (prefix) {
   const secureOrClientCertEnabled = castBool(getEnv(prefix, `PUBLIC_HTTPS`)) ||
     getEnv(prefix, `AUTH_CLIENT_CERT_ENABLED`)
 
-  const missingKeyOrCert = !getEnv(prefix, `TLS_KEY`) ||
-    !getEnv(prefix, `TLS_CERTIFICATE`)
-
-  if (secureOrClientCertEnabled && missingKeyOrCert) {
+  const tls = parseTLSEnv(prefix)
+  if (secureOrClientCertEnabled && _.isEmpty(tls)) {
     const _prefix = prefix ? prefix + '_' : ''
     throw new Error(
         `Missing ${_prefix}TLS_KEY or ${_prefix}TLS_CERTIFICATE`)
   }
+
+  _.forEach(tls, file => {
```

Comment:
If you're just going to throw an error anyway, `_.forEach(tls, fs.accessSync)`, is much cleaner.